### DISCUSSION TITLE
Plans to add Adetailer into Enfugue?

### DISCUSSION THREAD

**Discussion by pxlmover** [painebenjamin/app.enfugue.ai#101] [Ideas]
I'm very impressed by Enfugue, it's so easy to use compared to other UIs.  The only thing I'm really missing is ADetailer for hand and face enhancements/fixes.  Do you plan to add ADetailer into Enfugue by chance?

**Comment by painebenjamin** [comment]
Thank you so much for the kind words, it really means a lot to hear that. Releases can be a very demoralizing time when bugs slip through or the design just doesn't jive with people, but hearing from folks like yourself keeps me going.

You're like the third person to ask for ADetailer! I'm usually upscaling so it was never as high a priority for myself, but it's clear that it's an integral part of people's workflows so it's planned for 0.3.1 along with a small handful of other suggestions - they're detailed on [the 0.3.1 milestone](https://github.com/painebenjamin/app.enfugue.ai/milestone/9) which should be out sometime next week.

Thank you again, please don't hesitate to point out anything else you're missing!
